.travis.yml

```diff
@@ -0,0 +1,23 @@
+branches: {only: [master, develop]}
+sudo: false
+language: python
+python: 3.7
+cache: pip
+
+addons:
+  postgresql: 12
```

Comment:
wow, that is pretty advanced. I think city servers use version 10?

Reply:
Yeah, probably safer to downscale it a bit for now, guess I'll go with 11 :)